dist/json-refs.esm.js

```diff
@@ -0,0 +1 @@
+/Users/chris/oss/json-refs/src/index.js
```

Comment:
I'm not sure including this was intentional.

Reply:
The preconstruct build here was using the dev build instead of the production one - I've updated the committed file, but as suggested below, I think we should stop committing the dist/ folder.